front/pages/poke/[wId]/data_sources/[name]/index.tsx

```diff
@@ -382,99 +394,103 @@ const DataSourcePage = ({
+                        />
                       }
-                    />
-                  }
-                  action={
-                    <Button.List>
-                      <Button
-                        variant="secondary"
-                        icon={EyeIcon}
-                        onClick={() => {
-                          window.confirm(
-                            "Are you sure you want to access this sensible user data? (Access will be logged)"
-                          );
-                          void router.push(
-                            `/poke/${owner.sId}/data_sources/${
-                              dataSource.name
-                            }/view?documentId=${encodeURIComponent(
-                              d.document_id
-                            )}`
-                          );
-                        }}
-                        label="View"
-                        labelVisible={false}
-                      />
-                    </Button.List>
-                  }
-                >
-                  <ContextItem.Description>
-                    <div className="pt-2 text-sm text-element-700">
-                      {Math.floor(d.text_size / 1024)} kb,{" "}
-                      {timeAgoFrom(d.timestamp)} ago
-                    </div>
-                  </ContextItem.Description>
-                </ContextItem>
-              ))}
-            </ContextItem.List>
-            {documents.length == 0 ? (
-              <div className="mt-10 flex flex-col items-center justify-center text-sm text-gray-500">
-                <p>Empty</p>
-              </div>
-            ) : null}
+                      action={
+                        <Button.List>
+                          <Button
+                            variant="secondary"
+                            icon={EyeIcon}
+                            onClick={() =>
+                              onDisplayDocumentSource(d.document_id)
+                            }
+                            label="View"
+                            labelVisible={false}
+                          />
+                        </Button.List>
+                      }
+                    >
+                      <ContextItem.Description>
+                        <div className="pt-2 text-sm text-element-700">
+                          {Math.floor(d.text_size / 1024)} kb,{" "}
+                          {timeAgoFrom(d.timestamp)} ago
+                        </div>
+                      </ContextItem.Description>
+                    </ContextItem>
+                  ))}
+                </ContextItem.List>
+                {documents.length == 0 ? (
+                  <div className="mt-10 flex flex-col items-center justify-center text-sm text-gray-500">
+                    <p>Empty</p>
+                  </div>
+                ) : null}
+              </>
+            ) : (
+              <PokePermissionTree
+                owner={owner}
```

Comment:
permissionFilter should be set to `read` here no?

Reply:
why ?

Reply:
https://dust4ai.slack.com/archives/C050SM8NSPK/p1707400930486179